### PR TITLE
Assesment week 1 coding concepts.js

### PR DESCRIPTION
**Pull request by alappies**

### CODE REVIEW

**Comment by alappies** [comment]
Moment of clarity during class.

File: assesment-week-1-coding-concepts.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+// b) Verify and explain: because it is Counting JavaScript as the zeroth space and ruby would be index 1.
+
+
+// --------------------4) What will this log?
+
+// var weekendDays = ["saturday", "sunday"]
+// console.log(weekendDays.toUpperCase())
+
+// a) Your answer: It should show "SATURDAY" AND "SUNDAY"
+// b) Verify and explain: It is showing that .toUpperCase() is not a function when I run it. Either the upper case comand is not a recognised function in JavaScript or 'saturday' and 'sunday' are still in string form.
```
Comment (WalkerTexas-coder) [inline assesment-week-1-coding-concepts.js:47]: Close, the strings are currently encapsulated by an array. So to access those values we'll have to call on those indexes ourselves or use some kind of iteration to access them and run .toUpperCase() on them.

**Review by WalkerTexas-coder — APPROVED** [review]
just a small review for one file I'll send you a full review via slack

File: assesment-week-1-coding-concepts.js (hunk 1 above)
Comment (alappies) [reply assesment-week-1-coding-concepts.js:47]: To get into the plastic baggy 😎
It's slowly sinking in!

**Pull request merged**